app/controllers/api/v1/greetings_controller.rb

```diff
@@ -1,10 +1,12 @@
+# frozen_string_literal: true
```

Comment:
- [ ] It's not a good idea to disable rubocop rules, kindly enable this rule, and fix the linter error

Reply:
@descholar-ceo linters are a personal preference, sometimes they simply over-complicate functional code, they are not required. But I am appreciative of the review and have fixed all linter errors now.